guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc

```diff
@@ -1,7 +1,7 @@
 [id="Creating_an_Activation_Key_to_Consume_the_Content_View_{context}"]
 = Creating an activation key to consume the content view
 
-Use the following procedure to create an activation key that you can use to subscribe your hosts to content managed by {Project}.
+Create an activation key to assign various attributes to hosts during registration.
```

Comment:
This procedure seems to be specific to this workflow, and not general. I think we could actually just leave this sentence out.

Below, need to remove

```
. Click the *Subscriptions* tab on the Activation Key's details page.
. Click the *Add* tab.
. Select your new *CentOS Stream* product in the table.
. Click *Add Selected* to save.
```

Reply:
@jeremylenz Do you have any suggestions for a new opening sentence to replace `Create an activation key to assign various attributes to hosts during registration.` ?

Reply:
The procedure is guiding you through creating an AK so that your host consumes the content view, so I couldn't really think of anything to replace it that didn't just re-state the title. Do we have to have an opening sentence?

This entire chapter ("Basic Content Management Workflow") seems designed so that you follow all the procedures in order, rather than just picking out one. 

I guess we could say something like

```
Create another activation key to assign hosts to the content view you created in the previous procedure.
```

Reply:
Yeah we need an intro of some kind. I think what you said here works perfectly.